### PR TITLE
Add Polygon tetuBAL-80BAL-20WETH stable pool to the whitelist

### PR DESCRIPTION
**Pull request by AlehNat**
# Description

Please include a summary of the change and if relevant which issue is fixed. Please also include relevant motivation and context.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [x] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Pool should be visible at the balancer app

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@balancer-labs/frontend-v2",
-  "version": "1.53.8",
+  "version": "1.53.9",
```
Comment (garethfuller) [inline package.json:3]: The latest release is actually already [1.53.9](https://github.com/balancer-labs/frontend-v2/releases/tag/1.53.9). Please update to 1.53.10.

Comment (AlehNat) [reply package.json]: Updated

Comment (garethfuller) [reply package.json]: We've just released 1.53.11 now, so needs another bump to 1.53.12

**Comment by garethfuller** [comment]
I need to merge this before I do the weekly release so bumped the version again myself. Closing in favour of #1882.

**Pull request closed without merging**